src/components/common/Dot/Dots.module.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../../styles/mediaQueriesMixins.scss";
+@import "../../../styles/settings.scss";
+
+.dots {
+    position: absolute;
+    right: 0;
+    top: 0;
+
+    @include for-tablet {
+        position: relative;
+        margin: 0 auto;
+    }
+
+    @include for-phone {
+        top: 60px;
+    }
+
+    ul {
+        margin: 0;
+        padding: 0 0 0 1rem;
+        list-style: none;
+        background-color: $newFurniture-dots-ul-bg;
+        transform: translateY(15px);
+
+        li {
+            display: inline-block;
+            margin-left: 0.5rem;
+
+            a {
+                display: block;
+                width: 13px;
+                height: 13px;
+                border-radius: 6px;
+                background-color: $newFurniture-dots-bg;
+                font-size: 0;
+
+                &.active,
+                &:hover {
+                    @include bg-hover-animation;
+                    background-color: $newFurniture-active-dots-bg;
+                }
+            }
+        }
+    }
+}
+
+.hotDealsDots {
+    position: absolute;
+    right: 0;
+    top: 0;
+
+    ul {
+        margin-right: 0.5rem;
+        padding: 0 0 0 1rem;
+        list-style: none;
+        transform: translateY(15px);
+
+        li {
+            display: inline-block;
+            margin-left: 0.5rem;
+
+            a {
+                display: block;
```

Comment:
Wydaje mi się, że ten styl niczym nie różni się od tego wyżej więc nie jest potrzebny. To samo tyczy się ul i li